ee/clickhouse/queries/session_recordings/test/__snapshots__/test_clickhouse_session_recording_list.ambr

```diff
@@ -5,8 +5,7 @@
          any(session_recordings.start_time) as start_time,
          any(session_recordings.end_time) as end_time,
          any(session_recordings.duration) as duration,
-         any(session_recordings.distinct_id) as distinct_id,
-         any(first_pageview_event.pageview_properties) as first_pageview_event_properties ,
+         any(session_recordings.distinct_id) as distinct_id ,
          countIf(event = '$pageview') as count_event_match_0 ,
```

Comment:
Discussion point: What would it take to remove these fields and the event join from the query? If we could do an initial query on just the recordings table when you're looking at the list and then query each recording's events as necessary, it might be faster than trying to load a subset of events for all the recordings in the initial query.

Not a blocker

Reply:
This might already become a lot faster now that we have `event_summary` materialized columns. Ben and I will have a think on how we can iterate on this performance now that we have the columns in prod.